Labspt2-house-cup --Venky yagatilee

vy3191 opened the pull request:
Implementing Stripe Method - getting error/success msg from the back-end
Sending/Saving the success msg in db
Using the success msg stored in the db to implementing conditional rendering(still working...)

vy3191 committed:
Modified BillingPage.js

vy3191 committed:
Added image to stripe payment component
Modified AdminAnalyticsPage.js

vy3191 committed:
Modified checkout.js - Added axios.patch to change user payment details in the user's db
Modifed user_routes.js  -- Added user update details

vy3191 committed:
Modified AdminAnalyticsPage.js -- Added get method for getting users who are admin
Modified user_routes.js - Added get method for getting the user who are members
Modifed Checkout.js - Added extra functionality

louislee7 reviewed (approved):
reviewed and approved

vy3191 committed:
Modified App.js

vy3191 committed:
Fixed issues in user_middleware.js -- patch method
Fixed issues with getTokenFromAuth0 - middleware
Modified BillingPage and AdminAnalyticsPage.js
Solved SettingsPage.js functionality

vy3191 committed:
Modified AdminAnalyticsPage.js -- Implemented conditional rendering on stripe payment

vy3191 committed:
Resolved conflict issues
